lib/models/postgrest_error.dart

```diff
@@ -0,0 +1,51 @@
+// To parse this JSON data, do
+//
+//     final postgrestError = postgrestErrorFromJson(jsonString);
```

Comment:
Could you use dartdoc syntax /// to provide a description for this class.

where is `postgrestErrorFromJson`?

Reply:
I will add it and use dartdoc syntax too